corehq/apps/repeaters/models.py

```diff
@@ -412,6 +363,9 @@ class ShortFormRepeater(Repeater):
     version = StringProperty(default=V2, choices=LEGAL_VERSIONS)
     friendly_name = _("Forward Form Stubs")
 
+    class Formats(object):
+        generator = ShortFormRepeaterJsonPayloadGenerator
```

Comment:
this is the (simpler) config style for specifying a single format, since in that case you don't need to specify a slug, a display string, or a default.